utils/generic.py

```diff
@@ -232,6 +232,41 @@ def filter_low_conf_regions(mt: Union[hl.MatrixTable, hl.Table], filter_lcr: boo
     return mt
 
 
+def vep_or_lookup_vep(ht, reference_vep_ht, reference):
+    """
+    VEP a table, or lookup variants in a reference database
+
+    :param Table ht: Input Table
+    :param Table reference_vep_ht: A reference database with VEP annotations (must be in top-level `vep`)
+    :return: VEPped Table
+    :rtype: Table
+    """
+    from gnomad_hail.resources.basics import vep_config_path, context_ht_path
+
+    VEP_REFERENCES = {
+        'GRCh37': context_ht_path(),
+        'GRCh38': '',
```

Comment:
fully aware this is missing - i'll probably VEP something up today and put it in here